rootfs/ingest/bin/env.sh

```diff
@@ -1,3 +1,10 @@
+# Parameters for ingest scripts
+
+data_url="$DATA_URL"
+# Strip trailing slash
+data_url=$(echo $data_url | sed 's%\(.*[^/]\)/*%\1%')
+
 PORT="25080"
-REPL_URL="http://qserv-repl-ctl-0.qserv-repl-ctl:${PORT}"
+QUEUE_URL="mysql://qsingest:@qserv-ingest-db-0.qserv-ingest-db:3306/qservIngest"
 QSERV_INGEST_DIR="/ingest/data"
+REPL_URL="http://qserv-repl-ctl-0.qserv-repl-ctl:${PORT}"
```

Comment:
GitHub complains that: "No newline in the end of the file".

Reply:
:+1: